src/libraries/System.Windows.Extensions/src/System/Media/SoundPlayer.cs

```diff
@@ -261,16 +262,7 @@ private void LoadSync()
             // setup the http stream
             if (_uri != null && !_uri.IsFile && _stream == null)
             {
-#pragma warning disable SYSLIB0014 // WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead.
-                WebRequest webRequest = WebRequest.Create(_uri);
-#pragma warning restore SYSLIB0014
-                webRequest.Timeout = LoadTimeout;
```

Comment:
seems like we are loosing `LoadTimeout` without replacement. We may consider something like:
```c#
           using CancellationTokenSource cts = new CancellationTokenSource();
           cts.CancelAfter(LoadTimeout);
```
and pass it to `GetStreamAsync`